### ISSUE TITLE
Python nvtx.push_range function missing category argument

### ISSUE THREAD

**Issue by beru** [NVIDIA/NVTX#16]
https://github.com/NVIDIA/NVTX/blob/24a123e8b2c6bd498a27dbe867a96a8c01b0f583/python/nvtx/nvtx.py#L123

this should be

```py
def push_range(message=None, color="blue", domain=None, category=None):
```

since `category` is used here.

https://github.com/NVIDIA/NVTX/blob/24a123e8b2c6bd498a27dbe867a96a8c01b0f583/python/nvtx/nvtx.py#L152

**Comment by beru** [comment]
@shwina Please take a look.

**Comment by shwina** [comment]
Thanks for reporting! This should be fixed in nvtx 0.2.3 (installable via `pip`). Will close out this issue once a conda package is available too.

**Comment by beru** [comment]
Thank you for fixing the problem, team green. It's working well now.
